server/sonar-qa-util/src/main/java/org/sonarqube/qa/util/QProfileTester.java

```diff
@@ -49,44 +50,42 @@
     this.session = session;
   }
 
-  public QualityProfilesService service() {
+  public QualityprofilesService service() {
     return session.wsClient().qualityProfiles();
   }
 
   @SafeVarargs
-  public final QualityProfile createXooProfile(Organization organization, Consumer<CreateRequest.Builder>... populators) {
+  public final QualityProfile createXooProfile(Organization organization, Consumer<CreateRequest>... populators) {
     int id = ID_GENERATOR.getAndIncrement();
-    CreateRequest.Builder request = CreateRequest.builder()
-      .setOrganizationKey(organization.getKey())
+    CreateRequest request = new CreateRequest()
+      .setOrganization(organization.getKey())
       .setLanguage("xoo")
       .setName("Profile" + id);
     stream(populators).forEach(p -> p.accept(request));
-    return service().create(request.build()).getProfile();
+    return service().create(request).getProfile();
   }
 
   public QProfileTester activateRule(QualityProfile profile, String ruleKey) {
     return activateRule(profile.getKey(), ruleKey);
   }
 
   public QProfileTester activateRule(String profileKey, String ruleKey) {
-    ActivateRuleRequest request = ActivateRuleRequest.builder()
+    ActivateRuleRequest request = new ActivateRuleRequest()
       .setKey(profileKey)
-      .setRuleKey(ruleKey)
-      .build();
+      .setRule(ruleKey);
     service().activateRule(request);
     return this;
   }
 
   public QProfileTester deactivateRule(QualityProfile profile, String ruleKey) {
-    service().deactivateRule(profile.getKey(), ruleKey);
+    service().deactivateRule(new DeactivateRuleRequest().setKey(profile.getKey()).setRule(ruleKey));
     return this;
   }
 
   public QProfileTester assignQProfileToProject(QualityProfile profile, Project project) {
-    service().addProject(AddProjectRequest.builder()
-      .setProjectKey(project.getKey())
-      .setKey(profile.getKey())
-      .build());
+    service().addProject(new AddProjectRequest()
+      .setProject(project.getKey())
+      .setKey(profile.getKey()));
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this use of "setKey"; it is deprecated. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)